server/store/adapters/rdbms/dal/connection.go

```diff
@@ -161,14 +162,21 @@ func (c *connection) CreateModel(ctx context.Context, mm ...*dal.Model) (err err
 	c.mux.Lock()
 	defer c.mux.Unlock()
 	for _, m := range mm {
-		err = ddl.UpdateModel(ctx, c.dataDefiner, m)
+
+		if err != nil {
+			return
+		}
+		_, err = c.dataDefiner.TableLookup(ctx, m.Ident)
```

Comment:
This does not make sense. Probably error check after TableLookup call?

Reply:
Yeah you're right. There already is an error check after the TableLookup call. The check you noticed was an unnecessary no-op. Fixed.